### ISSUE TITLE
Healthcheck URL sending events that scribe doesn't like

### ISSUE THREAD

**Issue by vrivellino** [promotably/api#204]
I'm seeing these exceptions in the scribe logs in staging:

```
2015-04-01 17:44:06,678 [pool-1-thread-2] ERROR com.amazonaws.services.kinesis.clientlibrary.lib.worker.ProcessTask  - ShardId shardId-000000000000: Application processRecords() threw an exception when processing shard
org.postgresql.util.PSQLException: ERROR: null value in column "site_id" violates not-null constraint
  Detail: Failing row contains (98319, 84ab5ca6-e466-45c8-a632-79aeec69c7b1, session-start, null, null, a04bdceb-e5d1-46dc-8b88-bdb8d00c9e76, 8476c412-66a6-47a0-a594-7f9248edc2ea, 2015-04-01 17:44:06.671797+00, {"event-name":"session-start","event-format-version":"1","sessio..., null, f).
    at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2198)
    at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1927)
    at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:255)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:561)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:419)
    at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:365)
    at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
```

Along with:

```
2015-04-01 17:44:06,678 [pool-1-thread-2] ERROR com.amazonaws.services.kinesis.clientlibrary.lib.worker.ProcessTask  - ShardId shardId-000000000000: Skipping over the following data records: [{SequenceNumber: 49548491585947962684959762725197231911256919106849341442,Data: java.nio.HeapByteBuffer[pos=463 lim=463 cap=463],PartitionKey: 0f03c9db-b04b-4dfe-897d-214553f8863b}]
```

And:

``` json
{
    "thread_name": "pool-1-thread-2",
    "message": "Failed to write event  {:session_id #uuid \"8476c412-66a6-47a0-a594-7f9248edc2ea\", :type session-start, :control_group false, :event_id #uuid \"84ab5ca6-e466-45c8-a632-79aeec69c7b1\", :site_shopper_id nil, :shopper_id #uuid \"a04bdceb-e5d1-46dc-8b88-bdb8d00c9e76\", :site_id nil, :promo_id nil, :data #<PGobject {\"event-name\":\"session-start\",\"event-format-version\":\"1\",\"session-id\":\"8476c412-66a6-47a0-a594-7f9248edc2ea\",\"created-at\":\"2015-04-01T17:44:05.369Z\",\"shopper-id\":\"a04bdceb-e5d1-46dc-8b88-bdb8d00c9e76\",\"site-shopper-id\":null,\"request-headers\":{\"accept\":\"*/*\",\"connection\":\"keep-alive\",\"host\":\"10.236.3.245:3000\",\"user-agent\":\"ELB-HealthChecker/1.0\"}}>}",
    "@timestamp": "2015-04-01T17:44:06.677Z",
    "level": "INFO",
    "mdc": {},
    "file": "NO_SOURCE_FILE",
    "class": "clojure.tools.logging$eval1$fn__7",
    "line_number": "0",
    "logger_name": "scribe.event-consumer",
    "method": "invoke",
    "@version": 1,
    "source_host": "ip-10-236-154-56"
}
```

If you deep into the embedded message, you'll find `"user-agent":"ELB-HealthChecker/1.0"`

TL;DR: It looks like session-start events are getting triggered by the health-check URL. We should not attempt to send these events to Kinesis. (Or move this issue to scribe and have scribe ignore these events.)
